ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -29,7 +29,13 @@ private[platform] object CompletionFromTransaction {
   ): CompletionStreamResponse =
     CompletionStreamResponse.of(
       checkpoint = Some(toApiCheckpoint(recordTime, offset)),
-      completions = Seq(Completion.of(commandId, Some(OkStatus), transactionId)),
+      completions = Seq(
+        Completion().update(
```

Comment:
I'd prefer `Completion.of` with the relevant fields set to empty values, so it's explicit.

If you're very much against this, you can just do `Completion(commandId, OkStatus, transactionId)`, as that constructor takes default parameters.

Reply:
I agree about being explicit, considering that adding TODOs would be bad practice.

Reply:
I think it might be worth also considering disabling adding defaults to the constructor from `scalapb`. That way if you add a new field you have to explictly add it everywhere. And in tests you can always use `Type.defaultInstance`

Reply:
> I think it might be worth also considering disabling adding defaults to the constructor from scalapb

I agree it's a good idea but for a separate PR.